app/assets/stylesheets/pageflow/themes/default/navigation_share_box.scss

```diff
@@ -0,0 +1,43 @@
+.navigation_share_box {
+  padding: 15px;
+  text-align: center;
+
+  .share_box_icons {
+    display: flex;
+    text-align: center;
+
+    .menu_item {
+      min-height: 50px;
+      margin: 0 auto;
+    }
+
+    a.share.whats_app,
+    a.share.telegram {
+      display: none;
+    }
```

Comment:
iPad uses mobile navigation in landscape mode and thus offers different sharing options based on orientation. I'd suggest relying on `has_phone_platform` CSS class instead.